spec/features/user/user_can_filter_orders_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.feature 'user can view thier orders' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.